README.md

```diff
@@ -1 +1 @@
-# decodeRb
\ No newline at end of file
+## decodeRb
```

Comment:
- [ ] please include a descriptive readme file as required by Microverse, you can use [this template ](https://github.com/microverseinc/readme-template)as an example to create your own readme file.